src/ctapipe/utils/astro.py

```diff
@@ -4,63 +4,197 @@
 not provided by external packages and/or to satisfy particular needs of
 usage within ctapipe.
 """
-from astropy import units as u
+
+import logging
+from copy import deepcopy
+from enum import Enum
+
+import astropy.units as u
 from astropy.coordinates import Angle, SkyCoord
+from astropy.time import Time
+
+log = logging.getLogger("main")
+
+__all__ = ["get_star_catalog", "get_bright_stars"]
 
-__all__ = ["get_bright_stars"]
 
+class StarCatalogues(Enum):
+    Yale = {
+        "directory": "V/50/catalog",
+        "coordinates": {
+            "frame": "icrs",
+            "epoch": "J2000.0",
+            "RA": {"column": "RAJ2000", "unit": "h:m:s"},
+            "DE": {"column": "DEJ2000", "unit": "d:m:s"},
+            "pmRA": {"column": "pmRA", "unit": "arsec/yr"},
+            "pmDE": {"column": "pmDE", "unit": "arsec/yr"},
+        },
+        #: Vmag is mandatory (used for initial magnitude cut)
+        "columns": ["RAJ2000", "DEJ2000", "pmRA", "pmDE", "Vmag", "HR"],
+        "record": "yale_bright_star_catalog",
+    }  #: Yale bright star catalogue
+    Hipparcos = {
+        "directory": "I/239/hip_main",
+        "coordinates": {
+            "frame": "icrs",
+            "epoch": "J1991.25",
+            "RA": {"column": "RAICRS", "unit": "deg"},
+            "DE": {"column": "DEICRS", "unit": "deg"},
+            "pmRA": {"column": "pmRA", "unit": "mas/yr"},
+            "pmDE": {"column": "pmDE", "unit": "mas/yr"},
```

Comment:
The units should be already included in the table you get from astroquery, no need to hardcore them here

Reply:
not for hms (or at least not for Yale BSC):
```
<Column name='pmRA' dtype='float32' unit='arcsec / yr' format='{:6.3f}' description='?Annual proper motion in RA J2000, FK5 system' length=9096>
```

Reply:
That's because it's a string, not a value with a unit